### ISSUE TITLE
A11y_AzureDataStudio(Schema,Dacpac,Backup)_ImportBacpac_ImportBacpacSettings_ScreenReader:The selected file's location is not announced.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#6899]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
Application Name: Azure Data Studio
Application Version:1.10.0-insider (user setup)
Commit: 1ff9a2e
Electron: 4.2.7
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
Tools: Voiceover
MAC 10.14.6 
MAS References:MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Dacpac extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Data-tier application wizard from Context menu.
8. Select third radio button.
9. Navigate till next button & hit enter.
10. Start screen reader as voiceover.
11. Navigate using tab key to the "..." button file location and select a file.

**Actual:**
On selecting a file, the newly added file location is not announced by the screen reader , the announcement is just the dialog and the button name.

**Similar issue is observed on the following page:**
1.ExtractDacpacSettings:
Follow the exact above steps but select radio button 2.

**Expected:**
The file location of the new file added should be announced as soon as it has been added in the edit box.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users will not know the location of the newly added file as soon as it is added.

**MAS Reference:**
MAS1.3.1:https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
